include/z64item.h

```diff
@@ -303,6 +304,7 @@ typedef enum {
     /* 0x50 */ GI_50 = 0x50,
     /* 0x52 */ GI_SCALE_GOLD = 0x52, // Assumed, used in En_Fishing
     /* 0x59 */ GI_BOTTLE_POTION_RED = 0x59,
+    /* 0x5A */ GI_5A = 0x5A,
     /* 0x5B */ GI_POTION_RED = 0x5B,
```

Comment:
```suggestion
    /* 0x59 */ GI_BOTTLE_POTION_RED = 0x59,
    /* 0x5A */ GI_5A,
    /* 0x5B */ GI_POTION_RED,
```
Nice work with the enums. Note enum values to do not need to be explicitly declared when they are sequential and makes it easier to maintain the enum overall.